packaging/rpm/flightctl.spec

```diff
@@ -47,6 +47,10 @@ install -m 0755 packaging/greenboot/flightctl-agent-running-check.sh %{buildroot
 cp bin/flightctl-agent %{buildroot}/usr/bin
 cp packaging/must-gather/flightctl-must-gather %{buildroot}/usr/bin
 cp packaging/systemd/flightctl-agent.service %{buildroot}/usr/lib/systemd/system
+if [[ -d /etc/bash_completion.d ]]; then
+	bin/flightctl completion > /tmp/flightctl-completion.sh
+	sudo mv /tmp/flightctl-completion.sh /etc/bash_completion.d
```

Comment:
We should put the files in the %{buildroot} directory, which will be packaged later in the rpm; see the link from Fedora project to find the right places (we have macro directories for that)

Also bash completions for system installed packages should not go to `/etc/` (that is for user installed, manually managed files).

See the documentation here. Once you have the .sh file in tmp, follow the instructions for the install script and the %files section.

https://docs.fedoraproject.org/ca/packaging-guidelines/ShellCompletions/